src/sentry/api/endpoints/project_dynamic_sampling.py

```diff
@@ -85,25 +86,34 @@ def __run_discover_query(columns, query: str, params, limit, referrer: str, orde
             referrer=referrer,
         )["data"]
 
-    def __get_root_transactions_count(self, project, query, sample_size, query_time_range):
-        # Run query that gets total count of transactions with these conditions for the specified
-        # time period
+    def __get_transactions_count(
+        self,
+        project,
+        sample_size,
+        query,
+        query_time_range,
+        extra_query_args=("event.type:transaction",),
```

Comment:
What is the point of this argument?

Reply:
original ideea was to reuse this function, but i'll remove it.